sdk/couchbase-core/src/cbconfig/mod.rs

```diff
@@ -136,3 +136,36 @@ pub struct TerseConfig {
     #[serde(alias = "clusterCapabilities")]
     pub cluster_capabilities: HashMap<String, Vec<String>>,
 }
+
+#[derive(Deserialize, Debug, Clone)]
+#[non_exhaustive]
+pub struct CollectionManifestCollection {
+    #[serde(rename = "uid")]
```

Comment:
Do we need this for uid, name and history?

Reply:
Probably not, but it doesn't hurt to be explicit